loan/README.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Loan
+category: Behavioral
+language: en
+tag:
+- Financial Management
```

Comment:
Let's try to stick with the existing tags, if possible. See https://github.com/iluwatar/java-design-patterns/wiki/07.-Categories-and-Tags